changelogs/unreleased/4597-tsaarni-minor.md

```diff
@@ -0,0 +1,4 @@
+## Overload Manager
```

Comment:
I'd bump this changelog up to "major", it will definitely be one of the big features in the release :)